src/core/configs/authentication.config.ts

```diff
@@ -0,0 +1,4 @@
+export const authenticationConfig = {
+    secret: 'a1f68fbefc78865',
```

Comment:
it must be moved to `.env` as it is sensitive information and can not be stored like this.